src/jquery.multiselect.js

```diff
@@ -19,6 +19,12 @@
  *
  */
 (function($, undefined) {
+  // http://stackoverflow.com/a/3640212/401636
+  // custom css filter ex: :css(display=none)
+  $.expr[':'].css = function(obj, index, meta, stack) {
```

Comment:
I'm not fond of modifying anything in jQuery for this widget. This is a side effect that could impact other defined expressions in other widgets or by the user.

You should declare a method in the widget instead and use that as a parameter to filter.